### PR TITLE
Add "Debug Cell"

### PR DESCRIPTION
**Pull request by roblourens**
For microsoft/vscode#130524

And #1652

- Add a "Debug Cell" command
- Remove the "Debug" button from the toolbar

A little flakier than RBL, I keep having this issue where I send the 'disconnect' request but the Debug Session does not actually terminate, seen that before? Seems to work fine for RBL.

### CODE REVIEW

**Review by rchiodo — APPROVED** [review]

**Comment by DavidKutu** [comment]
The Lint test is failing because you need to run prettier on:
src/client/debugger/jupyter/debuggingManager.ts
src/client/debugger/jupyter/kernelDebugAdapter.ts

**Review by IanMatthewHuff — APPROVED** [review]

File: src/client/debugger/jupyter/debuggingManager.ts [hunk 1]
```diff
@@ -234,22 +244,40 @@ export class DebuggingManager implements IExtensionSingleActivationService, IDeb
...
-    private async startDebugging(doc: NotebookDocument, cell?: NotebookCell, options?: DebugSessionOptions) {
+    private async startDebuggingCell(doc: NotebookDocument, mode: KernelDebugMode.Cell | KernelDebugMode.RunByLine, cell: NotebookCell) {
+        const config: IKernelDebugAdapterConfig = {
+            type: pythonKernelDebugAdapter,
+            name: path.basename(doc.uri.toString()),
+            request: 'attach',
+            internalConsoleOptions: 'neverOpen',
+            justMyCode: true,
+            // add the doc uri to the config
+            __document: doc.uri.toString(),
```
Comment (roblourens) [inline src/client/debugger/jupyter/debuggingManager.ts:255]: @DavidKutu is this used anywhere? I couldn't actually find a usage

**Comment by roblourens** [comment]
> I keep having this issue where I send the 'disconnect' request but the Debug Session does not actually terminate, seen that before? Seems to work fine for RBL.

@DavidKutu any idea about that?

File: src/client/debugger/jupyter/debuggingManager.ts (hunk 1 above)
Comment (DavidKutu) [reply src/client/debugger/jupyter/debuggingManager.ts:270]: Yeah, I guess I removed it. We send the NotebookDocument as a parameter anyway. I'll remove it on my next PR.

**Comment by DavidKutu** [comment]
I don't see how you're sending the disconnect. Is it with the normal debugging widget?
I haven't had issues with neither normal debugging nor run by line

**Comment by roblourens** [comment]
The `disconnect` method gets called in the same way it is for RBL when the cell execution is finished. If I click the disconnect button, it works correctly

**Pull request merged**